R/get-datasets.R

```diff
@@ -169,6 +169,33 @@ loadDatasetFromURL <- function(url) {
     return(dataset)
 }
 
+datasetURLFromPath <- function(path) {
```

Comment:
This should be removed entirely, but I've left it (moved here from a file that was otherwise deleted) for the moment.